Handwriting/HandwritingIntentCSSValue.md

```diff
@@ -88,21 +88,27 @@ Authors are used to the [recommended practice of adding touch-action: none](http
 
 ### Keywords and States
 
+ The `touch-action` attribute accepts the following keywords:
+> Value: `auto` | `none` | [ [ `pan-x` | `pan-left` | `pan-right` ] | [ `pan-y` | `pan-up` | `pan-down` ] ] | `manipulation`
```

Comment:
Missing `pinch-zoom`, and the new `handwriting` keywords.

Reply:
I meant this more as a "currently accepts the following keywords:". I'll update the wording.

For pinch-zoom, I copied the list from the touch-action w3c link we have in the introduction section:
https://w3c.github.io/pointerevents/#the-touch-action-css-property
Which doesn't have pinch zoom. Is the link to an old version of the spec and I should update it?
Looking at the file on [github](https://github.com/w3c/pointerevents/blob/gh-pages/index.html), it seems like pinch-zoom is not coded into the spec, it's just mentioned in regards to manipulation. Should the spec be updated?

Reply:
I think the w3c spec touch-action values section is incomplete and just missing `pinch-zoom`.
`pinch-zoom` is mentioned under `manipulation`, but for some reason wasn't included as a keyword. I believe this is likely in error.
The spec should probably have a separate PR to include that keyword as well at some point.